lib/rubocop/cop/style/frozen_string_literal_comment.rb

```diff
@@ -41,29 +41,62 @@ module Style
       #   module Baz
       #     # ...
       #   end
+      # @example EnforcedStyle: always_true
+      #   # The `always_true` style enforces that the frozen string literal
+      #   # comment is set to `true`. This is a stricter option than `always`
+      #   # and forces projects to use frozen string literals.
+      #   # bad
+      #   # frozen_string_literal: false
+      #
+      #   module Baz
+      #     # ...
+      #   end
+      #
+      #   # bad
+      #   module Baz
+      #     # ...
+      #   end
+      #
+      #   # good
+      #   # frozen_string_literal: true
+      #
+      #   module Bar
+      #     # ...
+      #   end
       class FrozenStringLiteralComment < Cop
         include ConfigurableEnforcedStyle
         include FrozenStringLiteral
         include RangeHelp
 
         MSG = 'Missing magic comment `# frozen_string_literal: true`.'
```

Comment:
Should we change this one since we don't strictly require `# frozen_string_literal: true` ?

Reply:
Hrrmm -- I see how the current comment could be misleading, but I'm not sure what a better solution is. Maybe if we made it clear that we weren't giving the full text of the comment, e.g. `Missing magic "frozen_string_literal" comment`? But I think there's some value in giving the exact syntax expected for the comment.

Do you have thoughts on what to replace this with?

Reply:
> But I think there's some value in giving the exact syntax expected for the comment

If you're going to give exact syntax, then you should give all of the accepted values for clarity to the end user.  Otherwise, I would say `Missing magic "frozen_string_literal" comment`

Reply:
Yeah, probably there should be different messages for the different enforced styles. The current message is appropriate only for the new `always_true` style.

Reply:
Addressed in https://github.com/rubocop-hq/rubocop/pull/7634